other/hell/souls

```diff
@@ -0,0 +1,10 @@
+**Souls** | Additional Info - Hell Team
+Members of the hell team that can use Souls may use a Soul to protect someone from all attacks for the current night. If used during the day the player will be protected for the next night.
+Using a Soul to protect a player is an immediate effect.
+
+Members of the hell team that can use Souls may use a Soul to kill a player. The player will be killed immediately and it is not revealed why they died. 
```

Comment:
It mentions attacking someone when they sell their soul but here it mentions killing someone. i think it should be the former? Also is it a good idea to mention a failing message if it indeed is the former?

Reply:
Yeah it should be attacking. There is no failing message for the recipient though.

Reply:
Misread for what this is. Ignore the part about recipient.